bench/utils/__init__.py

```diff
@@ -420,10 +421,14 @@ def fetch_details_from_tag(_tag: str) -> Tuple[str, str, str]:
 	return org, repo, tag
 
 
-def is_git_url(url: str) -> bool:
-	# modified to allow without the tailing .git from https://github.com/jonschlinkert/is-git-url.git
-	pattern = r"(?:git|ssh|https?|\w*@[-\w.]+):(\/\/)?(.*?)(\.git)?(\/?|\#[-\d\w._]+?)$"
-	return bool(re.match(pattern, url))
+def check_or_get_parsed_git_obj_from_url(url: str) -> Any:
```

Comment:
`Any` is too broad. I think you can either get rid of the return type entirely, or use something like `Union[giturlparse.parse.Parsed, False]`